Packs/Base/Scripts/SanePdfReport/SanePdfReport_test.py

```diff
@@ -0,0 +1,52 @@
+import demistomock as demisto
+from SanePdfReport import *
+import subprocess
+import os
+import logging
+
+# disable warning from urllib3. these are emitted when python driver can't connect to chrome yet
+logging.getLogger("urllib3").setLevel(logging.ERROR)
```

Comment:
please remove the lines taken from rasterize

Reply:
done